### ISSUE TITLE
[Feature]: Modular CSS is not detected by Intellisense

### ISSUE THREAD

**Issue by NashJames** [microsoft/TypeScript#52693]
Does this issue occur when all extensions are disabled?: Yes

- VS Code Version: 1.7.4
- OS Version: WSL: Ubuntu-18.04

Steps to Reproduce:

1.  Create a TypeScript file and a `test.module.scss` file
2. Try import the css as `import styles from 'test.module.scss'`

You don't need it, but here's a basic example of Modular CSS usage:
```ts
import styles from './AppHeader.module.scss'

export default function AppHeader() {
  return <div className={styles.customDiv}></div>
}

```

The biggest annoyance is when Intellisense gets confused and tries to use one of the suggested things instead so you have to write it all out again.

**Comment by NashJames** [comment]
Closing. I setup a new laptop due to a career move and I noticed the enhancement suggested here in-use. Perhaps there has been an update or that laptop's configuration was changed somehow. Either way, I don't think this needs resources at the moment.

The only noticeable difference is the new system is MacOS, not Windows WSL. Perhaps that will be relevant/helpful if it does need investigating in future.
